tests/bgp/test_bgp_gr_helper.py

```diff
@@ -104,7 +104,7 @@ def _verify_prefix_counters_from_neighbor_after_graceful_restart(duthost, bgp_ne
 
     config_facts = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
     bgp_neighbors = config_facts.get('BGP_NEIGHBOR', {})
-    portchannels = config_facts.get('PORTCHANNEL', {})
```

Comment:
Likely some change in the image changed the naming convention?
Do you know if other test modules need such naming changes?

Reply:
It was caused by sonic-net/sonic-buildimage#13660